packages/web/src/content/docs/docs/gitlab.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: GitLab
+description: Use opencode in GitLab issues and merge requests
+---
+
+opencode integrates with your GitLab workflow.
+Mention `@opencode` in a comment, and opencode will execute tasks within your GitLab CI pipeline.
+
+---
+
+## Features
+
+- **Triage issues**: Ask opencode to look into an issue and explain it to you.
+- **Fix and implement**: Ask opencode to fix an issue or implement a feature.
+  It will work create a new branch and raised a merge request with the changes.
+- **Secure**: opencode runs on your GitLab runners.
+
+---
+
+## Setup
+
+Refer to the [GitLab CLI agents docs](https://docs.gitlab.com/user/duo_agent_platform/agent_assistant/)
```

Comment:
Is linking to the docs sufficient or should we replicate them?

I don't like to duplicate as it becomes more challenging to keep them all up to date.

Reply:
Yeah linking makes sense. I'll leave the snippet you have as an example.